### PR TITLE
unify style

### PR DESCRIPTION
**Pull request by rueshyna**
Unify style of
- spinner: the spinner will all in center
- import wallet page and setting page

### CODE REVIEW

File: components/import/aliases.tsx [hunk 1]
```diff
@@ -169,18 +170,24 @@ function Aliases() {
...
-            </label>
-            <Field
-              disabled
-              component="input"
-              name="effectivePeriod"
-              value={formState?.effectivePeriod ?? "Unknown"}
-              className="rounded p-2"
-            />
-          </div>
+          {typeof formState?.effectivePeriod != "undefined" && (
```
Comment (JulesGuesnon) [inline components/import/aliases.tsx:173]: `!!formState?.effectivePeriod` should work and is smaller

File: components/signersForm.tsx [hunk 2]
```diff
@@ -250,6 +250,7 @@ const SignersForm: FC<{
                               {index === 0 ? "Owner Name" : ""}
                             </label>
                             <Field
+                              disabled={props.disabled ? true : null}
```
Comment (JulesGuesnon) [inline components/signersForm.tsx:253]: Is there a reason for those conditions? Like it looks like it's the same as doing; `if (boolean) {true} else {false}`

**Review by JulesGuesnon — COMMENTED** [review]
Some really really minor js syntax related stuff, otherwise LGTM!

File: components/signersForm.tsx (hunk 2 above)
Comment (rueshyna) [reply components/signersForm.tsx:253]: the type is `boolean | undefined`. It looks like not ok to set `disabled = undefined`

Comment (JulesGuesnon) [reply components/signersForm.tsx:253]: Well basically if you don't add the props `disabled` to `<Field />` by default it'll be undefined. So the type of `disabled` is `boolean | undefined` which is matching our `disabled` props. So imo it should work, but this is really some details, and I don't really want to bother you which such a thing as it's really quick, so I'll check myself

**Review by JulesGuesnon — COMMENTED** [review]
LGTM

**Review by JulesGuesnon — APPROVED** [review]

File: components/signersForm.tsx (hunk 2 above)
Comment (rueshyna) [reply components/signersForm.tsx]: Ah, I got it! Thanks!

**Pull request merged**